moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -36,6 +36,7 @@ const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
         <DashboardStatusModal
           projectId={projectId}
           projectName={projectName}
+          data={data}
```

Comment:
since the only property you care about in `data` is the current phase ID, it would be better to require that prop explicitly, eg `currentPhaseId` instead of the generic `data` blob.

also, the `DasboardView` also uses this component and needs to be updated in the same way.